Cargo.toml

```diff
@@ -18,7 +18,7 @@ httparse = "1.7"
 openssl = "0.10"
 metrics = "0.20.0"
 metrics-util = "0.14.0"
-pea2pea = "0.40"
+pea2pea = "0.41"
```

Comment:
hmm, not sure if would it be a good idea to have this in a separate `uplift` solo commit (in this PR of course)

Reply:
Well, this test will not work very well without 0.41 version. However I can put new PR when there will be 0.42 version (should be today) and it will only contain that one line to bump pea2pea from 0.40 to 0.42. 
So by now, I'm rolling back to 0.40.

Reply:
Sorry, due to couple merges rebasing now commit to the first place is somehow problematic. I'll keep in mind our talk and try to produce more linear PR next time :)